unstructured/chunking/title.py

```diff
@@ -121,31 +123,29 @@ def chunk_by_title(
     # ----------------------------------------------------------------
```

Comment:
Commenting here since I'm unable to comment on the part I want to.

I'm not really understanding the difference between `new_after_n_chars` and `max_characters` from the docstring description. Why would users use `new_after_n_chars` when they can just use `max_characters`? Are they intended to be used together for some specific use case?

Reply:
Well, I'm not sure i'm clear about the intended use case, but this one is plausible to me and the only one that's occurred to me so far:

* I (the library end-user) want chunks that are around 250 chars because that's currently optimal for my language model platform.
* However, I would rather exceed that window somewhat than split a chunk mid-text if I can do so within 300 chars.
* So I set `new_after_n_chars = 250`, which makes most of my chunks around that size, and I set `max_characters = 300`, which allows slightly larger elements to be chunked whole, but ensures I never get any chunks bigger than that even if it means splitting a chunk mid-text.
* I've found through experimentation that this gives me the best balance on my particular corpus of documents.

I'm not confident in this account, this is just the only plausible idea I've had, so I haven't presumed to update the docstring.

I could ask around on this though and consider updating the docstring in a subsequent PR if folks thought this was right. The documentation would have to be updated too I expect.